packages/frontend/src/router/index.js

```diff
@@ -83,9 +83,25 @@ const routes = [
             },
             component: () => import('../views/StreamMain.vue')
           },
+          {
+            path: 'globals/',
+            name: 'globals',
+            meta: {
+              title: 'Globals | Speckle'
+            },
+            component: () => import('../views/Globals.vue')
+          },
+          {
+            path: 'globals/:commitId',
+            name: 'previous globals',
+            meta: {
+              title: 'Globals | Speckle'
+            },
+            component: () => import('../views/Globals.vue')
+          },
           {
             path: 'branches/',
-            name: 'branchs',
```

Comment:
🤣

Reply:
been waiting for my opportunity to fix this 🥺